README.md

```
@@ -135,8 +135,8 @@ SOFTWARE.
 [keepchangelog]: http://keepachangelog.com/en/1.0.0/
 [license-shield]: https://img.shields.io/github/license/jpbede/aiotankerkoenig.svg
 [maintenance-shield]: https://img.shields.io/maintenance/yes/2025.svg
-[poetry-install]: https://python-poetry.org/docs/#installation
-[poetry]: https://python-poetry.org
+[uv]: https://docs.astral.sh/uv/
```

markdownlint-cli2 (0.21.0)

[warning] Link and image reference definitions should be needed
Unused link or image reference definition: "uv"

(MD053, link-image-reference-definitions)